packages/getMyPrescriptions/src/getMyPrescriptions.ts

```diff
@@ -91,7 +129,21 @@ export async function eventHandler(event: APIGatewayProxyEvent): Promise<APIGate
   }
 }
 
-export const handler = middy(lambdaHandler)
+export const handler = middy(stateMachineEventHandler)
```

Comment:
we should call this stateMachineEventHandler to be consistent with apiGatewayHandler (and change sam template to reflect this)

Reply:
Have called it `stateMachineHandler` to follow the pattern of the api one.

Reply:
Actually, it has to stay as `handler`. The only way to specify a custom handler from the state machine is to include it in the arn, but that then fails based on a regex validation.